packages/daemon/src/migrations/20200803063219-addDatasetProcess.ts

```diff
@@ -0,0 +1,25 @@
+'use strict';
+import {QueryInterface, DataTypes} from 'sequelize';
+
+module.exports = {
+  up: async (queryInterface: QueryInterface) => {
+    return queryInterface.sequelize.transaction(t => {
+      return Promise.all([
+        queryInterface.addColumn('pipelines', 'datasetProcess', {
+          type: DataTypes.STRING
+        }, { transaction: t }),
+        queryInterface.addColumn('pipelines', 'datasetProcessParams', {
+          type: DataTypes.STRING
+        }, { transaction: t })
+      ])
+    })
+  },
+
+  down: (queryInterface: QueryInterface) => {
+    return queryInterface.sequelize.transaction(t => {
+      return queryInterface.removeColumn('pipelines', 'datasetProcess', { transaction: t }).then(() => {
```

Comment:
Will the `removeColumn` function return a Promise?

Reply:
Yes it is a Promise

Reply:
So shall we use async/await instead?